build.gradle

```diff
@@ -271,6 +271,15 @@ dependencies {
   compile group: 'org.springframework', name: 'spring-tx', version: '5.2.7.RELEASE'
 
   compile group: 'org.postgresql', name: 'postgresql', version: '42.2.14'
+  implementation 'org.springframework.boot:spring-boot-starter-oauth2-client'
+  compile "org.springframework.security:spring-security-web"
+  compile "org.springframework.security:spring-security-config"
+  compile "org.springframework.boot:spring-boot-starter-oauth2-client"
+  compile "org.springframework.boot:spring-boot-starter-oauth2-resource-server"
+  implementation 'uk.gov.hmcts.reform:idam-client:1.5.4'
+  compile group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '7.9'
+  compile group: 'io.jsonwebtoken', name: 'jjwt', version: '0.9.1'
+
```

Comment:
oauth2-client is added 2 times. Can we have all the dependencies for compile only?